src/common/utils/validator.js

```diff
@@ -263,6 +263,18 @@ function validateChains(chains) {
   })
 }
 
+/**
+ * false if all values in array are equal, else true
+ * @param {Array} values
+ * @returns {Bool}
+ */
+function notSameValues(values) {
+  if(values.includes(null) || values.includes(undefined)) {
+    return true;
+  }
+  return !values.every(v => v.trim().toLowerCase() === values[0].trim().toLowerCase());
```

Comment:
As I said in the slack message, we should make an exception for zzzz. Will wait to see what does Adam say on this.